scripts/package_sign_tacos.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
```

Comment:
Recommend adding some best practice, stop on error settings here (before any comments).

I don't have my full notes to hand (still on loaner laptop) but a good minimum is `set -e`. Linking to https://stackoverflow.com/a/2871034 which covers some others .

This could/should be done AFTER this get's merged. I.e. we know this works so lets get it in as-is, but then immediately follow up with sanity check additions and testing that.